spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/abstractnn/AbstractModule.scala

```diff
@@ -168,6 +168,18 @@ abstract class AbstractModule[A <: Activity: ClassTag, B <: Activity: ClassTag,
     Array((this, forwardTime, backwardTime))
   }
 
+  /**
+   * Get the forward/backward cost time for the module or its submodules
+   * and group by module type.
+   * @return (module type name, forward time, backward time)
+   */
+  final def getTimesGroupByModuleType():
```

Comment:
when will use this function

Reply:
Breaking down the performance in a model.